ui/components/app/toast-master/toast-master.js

```diff
@@ -256,6 +265,44 @@ function SwitchedNetworkToast() {
   );
 }
 
+function BaseNetworkToast() {
+  const t = useI18nContext();
+  const dispatch = useDispatch();
+
+  const showBaseNetworkToast = useSelector(
+    (state) => state.metamask.preferences.showBaseNetworkToast,
+  );
+
+  return (
+    showBaseNetworkToast && (
+      <Toast
+        key="switched-network-toast"
+        startAdornment={
+          <AvatarNetwork
+            size={AvatarAccountSize.Md}
+            borderColor={BorderColor.transparent}
+            src={BASE_TOKEN_IMAGE_URL}
+            name={BASE_DISPLAY_NAME}
+          />
+        }
+        text={t('baseNetworkToastMessage')}
+        actionText={t('seeDetails')}
+        onActionClick={() => {
+          dispatch(toggleNetworkMenu());
+          dispatch(
+            setEditedNetwork({
+              chainId: CHAIN_IDS.BASE,
+              nickname: BASE_DISPLAY_NAME,
+            }),
+          );
+        }}
+        hasLink={false}
+        onClose={() => dispatch(setShowBaseNetworkToast(false))}
```

Comment:
Should we also call `dispatch(setShowBaseNetworkToast(false))` in `onActionClick`? Otherwise, if the user clicks the close icon, the toast will be shown again, right?

Reply:
done [cde651e](https://github.com/MetaMask/metamask-extension/pull/28895/commits/cde651ed254e8c2aa09cb1293a522bbf2892d17c)